replica/table.cc

```diff
@@ -3199,7 +3199,7 @@ future<> compaction_group::cleanup() {
     co_await _t._cache.invalidate(std::move(updater), p_range);
 }
 
-future<> table::cleanup_tablet(locator::tablet_id tid) {
+future<> table::cleanup_tablet(database& db, db::system_keyspace& sys_ks, locator::tablet_id tid) {
```

Comment:
what's db used for? (I don't it being used in the patch that changes the signature.  maybe I missed something)

Reply:
Right, it's a leftover from a previous iteration of the patch, which was taking the current commitlog position from `db.commitlog()`. The current iteration uses `table::_highest_rp`, so it doesn't need `db`.

Reply:
so let's remove it